map_claims_test.go

```diff
@@ -66,3 +66,37 @@ func TestVerifyAud(t *testing.T) {
 		})
 	}
 }
+
+func Test_mapclaims_verify_issued_at_invalid_type_string(t *testing.T) {
```

Comment:
Could you follow the TestMapClaims etc. camel case style for tests please?

Reply:
For sure! I tried to keep the patch close to the original one 😅 (perhaps it could've been included in the test table of the existing test, but that required me to dive in deeper, so kept that for "future")

Reply:
Yes, adding it to the test matrix would of course be the ideal case. Let me see whether I can spin up a patch for this quickly.